README.md

```diff
@@ -164,6 +164,19 @@ defmodule MyApp.MyController do
 end
 ```
 
+When permissions' sets are specified through a `:one_of` map, the token is searched for at least one
+matching permissions set to allow the request. The fist set that matches will allow the request.
```

Comment:
@doomspork lol, yes and sorry about the typo, I'll fix that when I get to it. Thank you :)